test/fetch/util.js

```diff
@@ -119,18 +119,32 @@ test('sameOrigin', async (t) => {
 })
 
 test('isURLPotentiallyTrustworthy', (t) => {
-  const valid = ['http://127.0.0.1', 'http://localhost.localhost',
-    'http://[::1]', 'http://adb.localhost', 'https://something.com', 'wss://hello.com',
-    'file:///link/to/file.txt', 'data:text/plain;base64,randomstring', 'about:blank', 'about:srcdoc']
-  const invalid = ['http://121.3.4.5:55', 'null:8080', 'something:8080']
+  const valid = [
+    'http://127.0.0.1',
+    'http://[::1]',
+    'https://something.com',
+    'wss://hello.com',
+    'data:text/plain;base64,randomstring',
+    'about:blank',
+    'about:srcdoc',
+    'blob:http://example.com/550e8400-e29b-41d4-a716-446655440000'
```

Comment:
please add a test for something that ends with .localhost.

Reply:
done